### ISSUE TITLE
ideate: how to get customer feedback/do rapid customer development

### ISSUE THREAD

**Issue by ashumz** [waffleio/open-data-startup-challenge#31]
I ran across this the other day: http://www.smartchicagocollaborative.org/work/ecosystem/civic-user-testing-group/ - and they have a book about doing it in your own community http://www.cutgroupbook.org/2-origins/ -- "civic user testing" - One problem I noticed from CityCamp when people were doing their startup competition apps was that they were often targeting users they had no access too. That group aggregates citizens willing to test apps (sending them $5 visa gift cards initially). I imagine there's some type of matching of problem->solution for testers to try out civic tech oriented apps. Great way to get feedback from target audience.

Thinking about how do we do this at events like our's- can we bring in members of the community who are directly affected by our problem statements to the event for feedback?

**Comment by ashumz** [comment]
One idea (would love your feedback @homeyer as it fits in to the format of our event):

As we reach out to the community to share our themes and invite people to the event.... 

We can consider emphasize that we don't just need people to participate in the coding side -- but that each team ideally has a customer advisor or two who has felt the pain of the X problem statements we're suggesting or the general context. The customer advisors could help us flesh out the problem statement more prior to the event and become an integrated part of the team for giving feedback. 

i've often heard the suggestion for startups (mainly in lean startup methodology) to have a  customer advisory board, most often their early evangelists who share their vision and desire to solve the problem. this could be a great introduction to the value of that.

**Comment by ashumz** [comment]
from @jmeisel: To riff off the customer advisor idea, we could have a customer advisory board for the entire event that is a resource for all the teams.   Maybe they pop in for a a few hour chunk of time Saturday am but aren't needed for the whole event

**Comment by homeyer** [comment]
Yup, it'd be great to have a set of customers that show up to be available for interviews with the teams.

Love the idea too of getting the "customer persona" to be involved with the whole event, although then they turn into a team member and you still need to validate with other customers.

**Comment by ashumz** [comment]
Good point -- 

We can consider recruiting for both as an option. ie: we're looking for people with these problems to become integrated parts of teams to assist with usability and user experience (full event commitment), and we're looking for customer advisors to do customer development with from an 'outside the building' perspective (saturday morning commitment)
